app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -0,0 +1,224 @@
+ 
+### If you do not have a university degree
+
+If you do not have a degree, you can [apply to study a 3 to 4 year undergraduate course](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes) in England which combines a teaching degree with [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
+
+The [UK Council for International Student Affairs](https://www.ukcisa.org.uk/) has lots of helpful information about studying at an English university.
+
+### Further information
+
+[Train to be a teacher](/train-to-be-a-teacher) gives more information about training in England. 
+
+[Funding your training](/funding-your-training) explains what financial support is available if you are in the UK under the [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa), the [Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme) or the [Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme).
+
+## Tips for schools & training providers
+
+### Visas and immigration
+
+Most Ukrainian refugees will not need to be sponsored for a Skilled worker or Student visa. [Check the status of your visa](#check-the-status-of-your-visa) gives more information about visa schemes for Ukrainian refugees allowing them to live, work and study in the UK.
+
+### Qualifications
+
+Schools should assess the qualifications and experience of Ukrainian teachers in the same way as other overseas applicants, bearing in mind the exemption provided by the [4-year rule](https://www.gov.uk/guidance/recruit-teachers-from-overseas#employing-overseas-teachers-without-qts-the-4-year-rule).
+
+[UK ENIC](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) can provide a ‘statement of comparability’ to applicants showing how their academic and professional qualifications compare to the English equivalents. However, they will need scans or photographs of their original documents.
+
+### Safeguarding
+
+Schools must adopt robust recruitment procedures preventing overseas applicants who are unsuitable to work with children from securing a role in school.
+Schools must be guided by the safer recruitment policy outlined in [Keeping children safe in education.](https://www.gov.uk/government/publications/keeping-children-safe-in-education--2) They should also carry out additional checks related to international recruitment and training, outlined in [Recruit teachers from overseas](https://www.gov.uk/guidance/recruit-teachers-from-overseas#safeguarding-checks-for-teachers-from-overseas).  
```

Comment:
Change first sentence to:
Schools must be guided by [the statutory guidance on safeguarding children and safer recruitment](https://www.gov.uk/government/publications/keeping-children-safe-in-education--2).

Change 2nd sentence to:
They should also carry out [additional checks related to international recruitment and training](https://www.gov.uk/guidance/recruit-teachers-from-overseas#safeguarding-checks-for-teachers-from-overseas).